src/common/prometheus/prometheus.service.ts

```diff
@@ -199,25 +199,25 @@ export class PrometheusService implements OnApplicationBootstrap {
   public userValidators = this.getOrCreateMetric('Gauge', {
     name: METRIC_USER_VALIDATORS,
     help: 'User validators number',
-    labelNames: ['nos_name', 'status'],
+    labelNames: ['nos_id', 'nos_name', 'status'],
```

Comment:
```suggestion
    labelNames: ['nos_id', 'status'],
```
I suppose we can drop nos_name now and fetch it by ID

Reply:
No, we can't. Otherwise, we have to build additional queries to get the name from this identifier

Reply:
So adding ID will help to keep metrics consistent but dashboards will be a mess) Since we will have one id -> Many names

Reply:
> So adding ID will help to keep metrics consistent but dashboards will be a mess) Since we will have one id -> Many names

`sum by (nos_id) ethereum_validators_monitoring_user_validators`

Reply:
You're right. I need to spend a little more time creating a solution

Reply:
Reworked. See PR description